pennylane/ftqc/conditional_measure.py

```diff
@@ -94,10 +95,8 @@ def qnode(x, y):
         While such statements may not result in errors, they may result in
         incorrect behaviour.
     """
-    if qml.capture.enabled():
-        raise NotImplementedError(
-            "The `cond_measure` function is not compatible with program capture"
-        )
+    if capture.enabled():
+        cond(condition, true_fn, false_fn)
```

Comment:
Cannot comment above, but the docstring mentions not compatible with capture. This should no longer be the case, right?